packages/bar/src/types.ts

```diff
@@ -230,6 +233,8 @@ export type BarCommonProps<RawDatum> = {
 
     enableLabel: boolean
     label: PropertyAccessor<ComputedDatum<RawDatum>, string>
+    labelPosition: 'start' | 'center' | 'end'
```

Comment:
I remembered that we have a similar property for `legendsContinuousColorsLegendProps`, `titleAlign`, which uses `'start' | 'middle' | 'end'`, maybe we could use the same for consistency, what do you think?

Reply:
`'start' | 'middle' | 'end'` is used in quite a few places already, so it could make sense to factorize it a bit. Is there a specific way you would want to do that?

Reply:
For now, I think we can just use the same values, and handle the rest in another PR.